test/test-runtime-onMessage.js

```diff
@@ -239,4 +239,24 @@ describe("browser-polyfill", () => {
       });
     });
   });
+
+  it("ignores a certain chrome.runtime.lastError", () => {
+    const fakeChrome = {
+      runtime: {
+        lastError: {
+          message: "The message port closed before a response was received.",
+        },
+        sendMessage: sinon.stub(),
+      },
+    };
+
+    return setupTestDOMWindow(fakeChrome).then(window => {
+      fakeChrome.runtime.sendMessage
```

Comment:
Nit, do you mind to move this out of this callback? right after `const fakeChrome = {...};` at line 252.